recipes/restinio/all/conanfile.py

```diff
@@ -38,7 +38,7 @@ def layout(self):
 
     def requirements(self):
         self.requires("http_parser/2.9.4")
-        self.requires("fmt/10.1.1")
+        self.requires("fmt/11.0.1")
```

Comment:
```suggestion
        self.requires("fmt/10.2.1")
```

This is the most popular version by far in ConanCenter, I'd use this one to avoid conflicts